aria_data_loaders/aria_data_utils/episodic_memory_robotic_fetch.py

```diff
@@ -0,0 +1,435 @@
+import logging
```

Comment:
is this a main script to run the demo? if it is, then this should be move to some example folder

Reply:
Yes @jimmytyyang  this is the main driver node on PC that runs spot.
I didn't understand why it should be moved to another folder though. Can you elaborate on that please?

Reply:
I agree, this belongs in the utils/ folder.
However, we don't have a clean executables/ or examples/ folder.
We should make one and move this and the scripts for running Spot alone experiments there.

Reply:
Issue link - https://github.com/facebookresearch/spot-sim2real/issues/125